Code/src/main/java/nl/utwente/viskell/ghcj/GhciEvaluator.java

```diff
@@ -20,7 +20,7 @@ public GhciEvaluator() throws HaskellException {
 
     @Override
     protected List<String> getModules() {
-        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative");
+        return ImmutableList.of("Data.List", "Data.Maybe", "Data.Either", "Data.Monoid", "Control.Applicative", "Test.QuickCheck");
```

Comment:
It seems that, while `Test.QuickCheck` is a part of Haskell Platform, it's not distributed as part of the GHC you get from the Ubuntu repositories (the other ones are).

I don't really want to make this a preference - you wouldn't be able to open Viskell programs made in an environment where QC is available in one where it isn't.

Can we bundle QuickCheck somehow?